app/team/page.tsx

```diff
@@ -0,0 +1,115 @@
+import Head from 'next/head'
+
+import Image from 'next/image' // Import the 'Image' component
+
+import staffs from '@/data/staffs.json' // Import the JSON data
+
+export default function Team() {
+  const roles = [
+    'Developers',
+    'Admins',
+    'Translators',
+    'Contributors',
+    'Beta Testers'
+  ]
+
+  return (
+    <>
+      <Head>
+        <title>Would You - Team</title>
+      </Head>
+      <main className="">
+        <div className="flex flex-col px-8 xl:px-[17vw]">
+          <h1 className="mt-36 text-4xl font-bold text-white">
+            <span className="text-brand-red-100 drop-shadow-red-glow">
+              Meet{' '}
+            </span>{' '}
+            <span className="text-brand-blue-100 drop-shadow-blue-glow">
+              {' '}
+              the{' '}
+            </span>
+            Team
+          </h1>
+          <p className="text-neutral-500">
+            Meet the amazing team and contributors behind Would You.
+          </p>
+        </div>
+        <div className="mx-auto h-full max-w-5xl justify-center text-center">
+          {roles.map((role, i) => (
+            <>
+              <h2
+                key={role}
+                className="mt-10 select-none font-semibold text-neutral-300"
+              >
+                {role}
+              </h2>
+              <ul
+                key={i}
+                role="list"
+                className="text-pretty mx-auto mt-10 flex max-w-2xl flex-wrap justify-center gap-6 px-6 lg:mx-0 lg:max-w-none lg:gap-8 lg:px-8"
+              >
+                {staffs
+                  .filter((staff) => staff.categories.includes(role))
+                  .map((staff) => (
```

Comment:
An array reduce would be clean imo, but that's up to you if you actually want to do that 😄

Reply:
this is what they made on the website, I just added it to the page, I would structure the files in a different way but I won't change how Dominik made it.